torch/export/exported_program.py

```diff
@@ -342,13 +346,12 @@ def _decompose_and_get_gm_with_new_signature_constants(
 
     if ep.verifier.dialect == "TRAINING":
         mod = ep.module()
-        fake_args = []
-        for node in mod.graph.nodes:
-            if node.op == "placeholder":
-                fake_args.append(node.meta["val"])
-
-        fake_args_unwrapped = pytree.tree_unflatten(fake_args, mod._in_spec)
-        fake_mode = detect_fake_mode(fake_args)
+        fake_args = [
+            node.meta["val"] for node in mod.graph.nodes if node.op == "placeholder"
+        ]
+        fake_mode = torch._export.utils._detect_fake_mode_from_gm(mod)
+        if fake_mode is None:
+            fake_mode = FakeTensorMode(shape_env=ShapeEnv(), export=True)
```

Comment:
@ydwu4 when this is None? I feel if we have at least 1 call_function or at least 1 input node in the graph, this should be always present?

Reply:
It's this test:
```
def test_no_tensor_computation_3(self):
    class Module(torch.nn.Module):
        def forward(self, x, y):
            return 5
```
Do we even want to keep this test?